src/modules/file-share/create-volume/create-volume.js

```diff
@@ -0,0 +1,496 @@
+(function() {
+    "use strict";
+
+    var app = angular.module("TendrlModule");
+
+    app.controller("createVolumeController", createVolumeController);
+
+    /*@ngInject*/
+    function createVolumeController($scope, $state, $rootScope, $uibModal, $interval, config, nodeStore, utils, volumeCreationMapping, brickStore) {
+        var vm = this,
+            brickMappingTimer,
+            brickMappingCounter = 0;
+
+        vm.step = 1;
+        vm.settings = {};
+
+        vm.updateStep = updateStep;
```

Comment:
@cloudbehl Generally we follow the pattern of defining variables and then functions. Nothing wrong here, but its good to follow same pattern everywhere for consistency.